containers/runtime/xds/config/README.md

```diff
@@ -0,0 +1,37 @@
+# Configuration for xDS server
+
+Configuration JSON files are used to generate the xDS configuration for xDS server. The xDS server consumes these configuration and server these configuration to xDS client or Envoy sidecar.
```

Comment:
xDS server -> an xDS server. these -> this. server -> serves. xDS client -> an xDS client.